### PR TITLE
Admin settings panel enabled for all deployments

### PR DESCRIPTION
**Pull request by bion**
### Description

Enables the admin settings panel for all users and updates the automatic variable doc generation script to include each variable's mode.

#### General

Read the full guidelines for PRs [here](https://docs.civiform.us/contributor-guide/developer-guide/technical-contribution-guide#guidelines-for-pull-requests)

- [x] Added the correct label: < feature | bug | dependencies | infrastructure | ignore-for-release | database >
- [x] Created unit and/or browser tests which fail without the change (if possible)
- [x] Performed manual testing (Chrome and Firefox if it includes front-end changes)
- [x] Extended the README / documentation, if necessary

#### New Features

- [x] Add new FeatureFlag env vars to `server/conf/helper/feature-flags.conf`
- [x] Conditioned new functionality on a [FeatureFlag](https://docs.civiform.us/contributor-guide/developer-guide/feature-flags)
- [x] Wrote browser tests with the feature flag off and on, etc.

### Instructions for manual testing

If instructions are needed for manual testing by reviewers, include them here.

### Issue(s) this completes

Fixes #<issue_number>; Fixes #<issue_number>...

### CODE REVIEW

File: env-var-docs/generate_markdown.py [hunk 1]
```diff
@@ -160,6 +160,13 @@ def append_node_to_out(node: env_var_docs.parser.Node):
         if group is not None:
             desc = group.group_description
         if var is not None:
+            if var.mode == env_var_docs.parser.Mode.ADMIN_WRITEABLE:
```
Comment (rockycodes) [inline env-var-docs/generate_markdown.py:163]: suggestion (non-blocking): Please also update the readme [here](https://github.com/civiform/civiform/tree/main/env-var-docs/parser-package) to include a sentence about `ADMIN_WRITEABLE` under "mode".

File: env-var-docs/generate_markdown.py [hunk 2]
```diff
@@ -171,7 +178,7 @@ def append_node_to_out(node: env_var_docs.parser.Node):
             if var.values is not None:
                 out += "- Allowed values:\n"
                 for val in var.values:
-                    out += f"   - {val}\n"
+                    out += f"   - `{val}`\n"
```
Comment (rockycodes) [inline env-var-docs/generate_markdown.py:181]: question (non-blocking): just curious why we need the backticks. was it not printing correctly?

**Review by rockycodes — APPROVED** [review]
🎉

File: env-var-docs/generate_markdown.py (hunk 2 above)
Comment (bion) [reply env-var-docs/generate_markdown.py:181]: It prints fine without them, but the backticks format the values as code with a monospace font, making them a little easier to read to ensure each character gets copied correctly if someone copies and pastes a value

File: env-var-docs/generate_markdown.py (hunk 1 above)
Comment (bion) [reply env-var-docs/generate_markdown.py:163]: Good catch, thanks!

Comment (bion) [reply env-var-docs/generate_markdown.py:163]: Done

File: env-var-docs/generate_markdown.py (hunk 2 above)
Comment (rockycodes) [reply env-var-docs/generate_markdown.py:181]: nice!

**Pull request merged**